portfolio/src/main/webapp/script.js

```diff
@@ -49,10 +49,11 @@ function showComments() {
   const maxComment = sessionStorage.getItem('max-comment') || 1;
   document.getElementById('max-comment').value = maxComment;
   document.getElementById('comments').innerHTML = '';
-  
+  document.querySelector('.spinner-border').style.display = 'inline-block';
```

Comment:
Use a class instead of setting style directly.

```css
.spinner-border {
  display: inline-block;
}

.hidden {
  display: none;
}
```

then set "hidden" in JS

Reply:
I used classList.toggle instead. But it seems like it does not work for IE. lmk what you think about this.

Reply:
I wouldn't worry about browser compatibility, especially IE. If you do want to support IE I think only the second parameter of toggle isn't supported. See https://caniuse.com/#feat=classlist